RSignatureVerifier/RealSignatureVerifier.swift

```diff
@@ -0,0 +1,47 @@
+/// Signature Verifier Config public API functions
+internal final class RealSignatureVerifier {
+    static let shared = RealSignatureVerifier()
+
+    private let environment: Environment
+    private let apiClient: APIClientType
+    private var fetcher: Fetcher
+    private let keyStore: KeyStore
+    private let verifier: Verifier
+
+    init(environment: Environment = Environment(),
+         apiClient: APIClientType = APIClient(),
+         fetcher: Fetcher? = nil,
+         keyStore: KeyStore = KeyStore(),
+         verifier: Verifier = Verifier()) {
+
+        self.environment = environment
+        self.apiClient = apiClient
+        self.fetcher = fetcher ?? Fetcher(client: apiClient, environment: environment)
+        self.keyStore = keyStore
+        self.verifier = verifier
+    }
+
+    func verify(signature: String,
```

Comment:
I didn't know how to test this method without pushing any sensitive data

Reply:
I think we can do it like [remote config](https://github.com/rakutentech/ios-remote-config/blob/master/Tests/Unit/VerifierTests.swift#L6-L15) where we tested the verifier with known data.

Reply:
Ok. Actually that part is already here. I realised that I don't need to test this class against real data - I just need mocks

Reply:
oh sorry - I see it was added in a previous PR.